services/static-webserver/client/source/class/osparc/data/model/StudyUI.js

```diff
@@ -83,15 +83,17 @@ qx.Class.define("osparc.data.model.StudyUI", {
     },
 
     templateType: {
-      check: [null, "hypertool"],
+      check: [null, "hypertool", "tutorial", "template"],
       init: null,
       nullable: true,
       event: "changeTemplateType",
     },
   },
 
   statics: {
-    HYPERTOOL_TYPE: "hypertool",
+    TEMPLATE_TYPE: "TEMPLATE",
```

Comment:
I proposed matus that a possible name for the current template's "TEMPLATE" type is to call it in the API "BLUEPRINT". This way, a template project can be a "BLUEPRINT", "HYPERTOOL",  "TUTORIAL", etc